src/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -55,6 +55,18 @@ public function get_properties() {
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
+			'quantity_min'         => [
+				'description' => __( 'The minimum quantity than can be added to the cart at once.', 'woo-gutenberg-products-block' ),
+				'type'        => 'integer',
+				'context'     => [ 'view', 'edit' ],
+				'readonly'    => true,
+			],
+			'quantity_step'        => [
```

Comment:
I'm still on the fence about this property. You could argue for min and max (limit) in the API because it's something that can be validated against on add to cart requests, but the step is a UI restriction, not something you're using in the API. Also `step` is specific to HTML number inputs. Does it make sense here?

We could use `quantity_increment` and validate that the amount passed is an increment of that. e.g. increment of 2 would prevent a qty of 1 being accepted.

Thoughts? Too much?

Do you also think we should group these properties like `add_to_cart`?